doc/source/tutorial/optimize.rst

```diff
@@ -50,11 +50,7 @@ with the *Nelder-Mead* simplex algorithm (selected through the ``method``
 parameter):
 
     >>> import numpy as np
-    >>> from scipy.optimize import minimize
-
-    >>> def rosen(x):
-    ...     """The Rosenbrock function"""
-    ...     return sum(100.0*(x[1:]-x[:-1]**2.0)**2.0 + (1-x[:-1])**2.0)
+    >>> from scipy.optimize import minimize, rosen
```

Comment:
FWIW the original is helpful in showing how to define a user function, which using the black-box `scipy.optimize.rosen` doesn't. So maybe better to keep the original and mention that there's a prefab equivalent.

Reply:
Happy to change, are you wanting all originals to be left or just one?

Reply:
I'm pretty sure I'll take a look at the originals next time I need to write a function to `minimize`, even moreso if I need its hessian. So yeah, all of them are useful, more than the fact that `rosen_hess` can be imported from scipy.optimize.